executor/executor_test.go

```diff
@@ -475,6 +475,63 @@ func TestTimeout(t *testing.T) {
 	}
 }
 
+func TestTeardownAbort(t *testing.T) {
```

Comment:
this test is to check if the user teardown step is running correct ? and not abort signal trap.

Reply:
added test for notify signal